snuba/datasets/configuration/entity_builder.py

```diff
@@ -136,8 +170,9 @@ def build_entity_from_config(file_path: str) -> PluggableEntity:
         required_time_column=config["required_time_column"],
         validators=_build_entity_validators(config["validators"]),
         translation_mappers=_build_entity_translation_mappers(
-            config["translation_mappers"]
+            config.get("translation_mappers", {})
```

Comment:
`translation_mappers` is not a required field in the schema, and in fact some entities (CDC) don't specify them. So I changed this to handle entities that don't specify any mappers.